sidekick/lib/src/commands/init_command.dart

```diff
@@ -117,9 +139,57 @@ class InitCommand extends Command {
       throw 'The CLI name $cliName is already taken by an executable on your system see $cliNameCollisions';
     }
 
-    print("\nGenerating ${cliName}_sidekick");
+    print('${white('cliName:', bold: false)} $cliName\n');
+
+    final packageDir = Directory(
+      argResults!['cliPackageDirectory'] as String? ??
+          () {
+            final Directory defaultDir = () {
+              final packagesDir = projectRoot.directory('packages');
+              if (packagesDir.existsSync()) {
+                return packagesDir;
+              } else {
+                return projectRoot;
+              }
+            }();
+
+            print(
+              '${green('cliPackageDirectory - Enter the directory in which the ${cliName}_sidekick CLI package should be created.')}\n'
+              '(absolute or relative to ${projectRoot.absolute.path})\n'
+              'I.e. directory `packages` in mono-repos,\n'
```

Comment:
`e.g.` (= latin for `exemplia gratia`, for example) should be used here because it's an example that most mono-repos contain a `packages` dir, but not all do. If all mono-repos did, `i.e.` (= latin for `id est`, that is) would be correct.

Taking latin in high school finally paid off :D
```suggestion
              'E.g. directory `packages` in mono-repos,\n'
```